chaoscenter/subscriber/go.mod

```diff
@@ -65,6 +63,7 @@ replace (
 	github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309
 
 	github.com/emicklei/go-restful => github.com/emicklei/go-restful v2.16.0+incompatible
+	github.com/litmuschaos/chaos-operator => ../../../chaos-operator
```

Comment:
This is using your local operator repo which isn't right

Reply:
According to discussion in contributor call this will be updated later once operator pr is merged.

Reply:
sure.

Reply:
PR to be merged: https://github.com/litmuschaos/chaos-operator/pull/475